### ISSUE TITLE
Improve resource claiming for Cartesian command handles

### ISSUE THREAD

**Issue by stefanscherzinger** [fzi-forschungszentrum-informatik/cartesian_ros_control#12]
The current implementation of Cartesian command handles allows to run joint-based and Cartesian-based ROS-controllers in parallel (see e.g. [here](https://github.com/UniversalRobots/Universal_Robots_ROS_Driver/issues/373#issuecomment-843352520)). While technically they do not claim the same resources and hence could run in parallel regarding the ROS-Control philosophy, some drivers go for an [either-or-approach](https://github.com/UniversalRobots/Universal_Robots_ROS_Driver/blob/beta-testing/ur_robot_driver/src/hardware_interface.cpp#L665).

Let's rethink resource handling for these cases.

**Comment by fmauch** [comment]
Closing in favor of https://github.com/UniversalRobots/Universal_Robots_ROS_passthrough_controllers/issues/1
